src/server/surviveler/world.go

```diff
@@ -180,26 +344,26 @@ func (w World) IntersectingTiles(bb d2.Rectangle) []*Tile {
  */
 func (w *World) AttachEntity(ent Entity) {
 	// retrieve list of tiles intersecting with the entity aabb
-	tileList := w.IntersectingTiles(ent.Rectangle())
+	//tileList := w.IntersectingTiles(ent.Rectangle())
 
 	// attach this entity to all those tiles
-	w.attachTo(ent, tileList...)
+	//w.attachTo(ent, tileList...)
 
 	// add those links to the world (for fast query by entity id)
-	w.Entities[ent.Id()] = tileList
+	//w.Entities[ent.Id()] = tileList
 }
 
 /*
  * DetachEntity detaches an entity from the underlying world representation
  */
 func (w *World) DetachEntity(ent Entity) {
 	// retrieve tile list for this entity
-	tileList := w.Entities[ent.Id()]
```

Comment:
Same here: should we remove those two functions? Is there any point in keeping them here?